app/models/spree/order.rb

```diff
@@ -138,10 +138,11 @@ def states
     }
 
     scope :with_line_items_variants_and_products_outer, lambda {
-      joins('LEFT OUTER JOIN spree_line_items ON (spree_line_items.order_id = spree_orders.id)')
-        .joins('LEFT OUTER JOIN spree_variants ' \
-               'ON (spree_variants.id = spree_line_items.variant_id)')
-        .joins('LEFT OUTER JOIN spree_products ON (spree_products.id = spree_variants.product_id)')
+      joins(<<~SQL.squish)
+        LEFT OUTER JOIN spree_line_items ON (spree_line_items.order_id = spree_orders.id)
+        LEFT OUTER JOIN spree_variants ON (spree_variants.id = spree_line_items.variant_id)
+        LEFT OUTER JOIN spree_products ON (spree_products.id = spree_variants.product_id)
+      SQL
```

Comment:
A multi-line string is fine here, possibly nicer to read. No need for `squish` but it doesn't matter.

Reply:
It's a style thing. Without the `squish`, the `SQL` on line 145 has to have no indentation, so it would appear all the way to left. It was bugging me.

In the future, should I leave it un-squished?

Reply:
It doesn't have to be on the left, you can have indentation in SQL statements. Anyway, it doesn't matter.